.circleci/config.yml

```diff
@@ -113,12 +113,25 @@ jobs:
           keys:
             - v1-dev-bundle-cache-{{ checksum "meteor" }}
             - v1-dev-bundle-cache-
+      - run:
+          name: Combine NPM Shrinkwrap Files
+          command: |
+            for d in packages/*/.npm/package; do cat $d/npm-shrinkwrap.json >> shrinkwraps.txt; done
```

Comment:
couldn't this just be `cat packages/*/.npm/package packages/*/.npm/plugin/* >shrinkwraps.txt` ?

Reply:
Yeah I think so!